source/install/desktop-msi-gpo.rst

```diff
@@ -5,7 +5,12 @@ Desktop MSI Installer and Group Policy Installation Guides (Beta)
 This guide provides steps to install the MSI and use Group Policies in Windows Pro or Enterprise. The MSI installer package can be downloaded `here <https://github.com/mattermost/desktop/releases/tag/v4.3.0>`_. 
 
 .. note::
-  The Mattermost MSI installer and Group Policy (GPO) definitions are in Beta with v4.3.0. If you are using this installer or GPOs and have feedback, please let us know in the `MSI Installer channel on our Community server <https://community.mattermost.com/core/channels/msi-installer>`_. 
+    The Mattermost MSI installer and Group Policy (GPO) definitions are in Beta. Before installing this Beta, please ensure you have reviewing the following known issues:
+
+    - `Clicking a desktop notification does not bring the app into focus <https://mattermost.atlassian.net/browse/MM-19511>`_
+    - `Clicking a desktop notification does not switch to the channel the notification came from <https://mattermost.atlassian.net/browse/MM-19510>`_ 
+
+    If you are using this installer or GPOs and have feedback, particularly organizations executing remote deployments, please contact us in the `MSI Installer channel on our Community server <https://community.mattermost.com/core/channels/msi-installer>`_. We hope to promote this installer out of Beta when the known issues are addressed and we are confident that the various deployment scenarios expected in production environments are tested sufficiently with the help of organizations using this Beta. Feedback is highly appreciated.
```

Comment:
```suggestion
    If you are using this installer or GPOs and have feedback, particularly organizations executing remote deployments, please contact us in the `MSI Installer channel on our Community server <https://community.mattermost.com/core/channels/msi-installer>`_ or on our `community forum <https://forum.mattermost.org/>`_. We hope to promote this installer out of Beta when the known issues are addressed and we are confident that the various deployment scenarios expected in production environments are tested sufficiently with the help of organizations using this Beta. Feedback is highly appreciated.
```

Reply:
Propose including forums as an option to share feedback since it supports GitHub SSO. (email/password login to community can be a barrier)